app/assets/stylesheets/components/sub_nav.scss

```diff
@@ -10,21 +9,36 @@
     padding-bottom: 10px;
   }
 
+  .nav-links {
+    display: flex;
+    justify-content: space-around;
+
+    @media (min-width: 640px) {
+      justify-content: flex-end;
+    }
+  }
+
   nav {
     text-align: right;
+    margin-top: 4px;
+
+    > :first-child {
+      margin-left: 0px;
+    }
 
     a {
       margin-left: 15px;
+      margin-bottom: 5px;
       padding-top: 10px;
       padding-bottom: 5px;
       display: inline-block;
       text-decoration: none;
-      border-bottom: 5px solid white;
       font-size: 16px;
     }
 
     .active {
       border-bottom: 5px solid black;
+      margin-bottom: 0px !important;
```

Comment:
This ensures if the link is active, it has 5px underline rather than the 5px margin which the rest have from L31. Ensures that all the subnav links line up.